website/blog/2022-04-08-adding-fonts.md

```diff
@@ -0,0 +1,195 @@
+- https://convertio.co/ttf-woff/
+- https://cloudconvert.com/ttf-to-woff
+- https://everythingfonts.com/ttf-to-woff
+
+Fonts should go inside the **_/assets/fonts_** folder inside your theme. Paste the fonts you want to use there. You will also notice that this folder contains an **_index.js_** file. This file is used to import fonts into your project. Here's an example of how I imported my fonts:
+```js
+// SourceSansPro WOFF
+import './SourceSansPro-Bold.woff';
+import './SourceSansPro-BoldItalic.woff';
+import './SourceSansPro-Italic.woff';
+import './SourceSansPro-Regular.woff';
+import './SourceSansPro-Light.woff';
+import './SourceSansPro-LightItalic.woff';
+
+// SourceSansPro WOFF2
+import './SourceSansPro-Bold.woff2';
+import './SourceSansPro-BoldItalic.woff2';
+import './SourceSansPro-Italic.woff2';
+import './SourceSansPro-Regular.woff2';
+import './SourceSansPro-Light.woff2';
+import './SourceSansPro-LightItalic.woff2';
+
+// NotoSerif WOFF
+import './NotoSerif-Bold.woff';
+import './NotoSerif-BoldItalic.woff';
+import './NotoSerif-Italic.woff';
+import './NotoSerif-Regular.woff';
+
+// NotoSerif WOFF2
+import './NotoSerif-Bold.woff2';
+import './NotoSerif-BoldItalic.woff2';
+import './NotoSerif-Italic.woff2';
+import './NotoSerif-Regular.woff2';
+```
+
+To add these fonts as your base font and secondary font, go to the global manifest located in **_/src/Blocks_** and add the following inside `globalVariables`:
+```json
+"globalVariables": {
+	//...
+	"baseFont": "SourceSansPro",
+	"secondaryFont": "NotoSerif",
+	//...
+}
+```
+
+Next, you can create a new file called **__typography.scss_** inside your **_/assets/styles/parts/utils/_** folder and add the following:
+
+```scss
+@include font-face(global-settings(baseFont), 'SourceSansPro-Light', 300);
+@include font-face(global-settings(baseFont), 'SourceSansPro-LightItalic', 300, italic);
+@include font-face(global-settings(baseFont), 'SourceSansPro-Regular', 400);
+@include font-face(global-settings(baseFont), 'SourceSansPro-Italic', 400, italic);
+@include font-face(global-settings(baseFont), 'SourceSansPro-Bold', 700);
+@include font-face(global-settings(baseFont), 'SourceSansPro-BoldItalic', 700, italic);
+
+@include font-face(global-settings(secondaryFont), 'NotoSerif-Bold', 700);
+@include font-face(global-settings(secondaryFont), 'NotoSerif-BoldItalic', 700, italic);
+@include font-face(global-settings(secondaryFont), 'NotoSerif-Italic', 400, italic);
+@include font-face(global-settings(secondaryFont), 'NotoSerif-Regular', 400);
+```
+
+Because this is a new file, we have to include it. You can do that inside **_/assets/styles/parts/_shared.scss_** file:
```

Comment:
Please escape the `_` in `_shared.scss` in case Docusaurus change their Markdown parser in the future.